app/client/views/dashboard/dashboard.html

```diff
@@ -1,6 +1,6 @@
 <div id="dashboard" class="page">
-  <div class="divided title">
-    Dashboard
+  <div class="divided title"> <!-- style should be in the css, cant find it -->
```

Comment:
its on the `_page.scss` file, in the `stylesheets` directory. Divided is not as important but should be kept for consistency.